.github/workflows/android-appcenter.yml

```diff
@@ -5,13 +5,19 @@ on:
   push:
     branches: [ master ]
     paths:
-      - 'demo/android/Activity/porcupine-activity-demo-app/build.gradle'
       - '.github/workflows/android-appcenter.yml'
+      - 'demo/android/Activity/porcupine-activity-demo-app/**'
+      - '!demo/android/Activity/porcupine-activity-demo-app/README.md'
```

Comment:
there's not a README at that location. I think the paths should be `demo/android/Activity/**` and `!demo/android/Activity/README.md`